projects/js-packages/components/components/admin-section/index.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+/**
+ * This is the wrapper component to build sections within your admin page.
+ *
+ * @param {object} props - Component properties.
+ * @returns {React.Component} JetpackAdminSection component.
+ */
+const JetpackAdminSection = props => {
+	const { children, bgColor } = props;
+	let mainClassName = 'jp-admin-section';
+	if ( bgColor === 'grey' ) {
+		mainClassName += ' jp-admin-section--bg-grey';
+	}
```

Comment:
You could start to use [classnames](https://www.npmjs.com/package/classnames) utility here.

```
const mainClassName = classNames(
  'jp-admin-section',
  { 'jp-admin-section--bg-grey': bgColor === 'grey' }
);
```
Looks like we already have it on the project: 

https://github.com/Automattic/jetpack/search?q=classnames

Reply:
thanks! Will look into it

Reply:
Done in https://github.com/Automattic/jetpack/pull/21826/commits/70c68739a422181451b358854e424b402ca4563f